Makefile

```diff
@@ -172,9 +172,12 @@ bundle: manifests kustomize ## Generate bundle manifests and metadata, then vali
 	$(KUSTOMIZE) build --load-restrictor LoadRestrictionsNone config/manifests | operator-sdk generate bundle -q --overwrite --version $(VERSION) $(BUNDLE_METADATA_OPTS)
 	operator-sdk bundle validate ./bundle
 
-.PHONY: image
-image: manager ## Build the operator image
+.PHONY: image-build
+image-build: manager ## Build the operator image
 	$(CONTAINER_ENGINE) build -t $(OPERATOR_IMAGE) .
+
+.PHONY: image-push
```

Comment:
I think we already have such command https://github.com/mongodb/mongodb-atlas-kubernetes/blob/075a11c2dd5b17ca6f8eec00c91db334d247b174/Makefile#L232